e2e/packages/client-vanilla/src/mud/getNetworkConfig.ts

```diff
@@ -28,7 +28,7 @@ export async function getNetworkConfig() {
     faucetServiceUrl: params.get("faucet") ?? chain.faucetUrl,
     worldAddress,
     initialBlockNumber,
-    indexerUrl: params.get("indexerUrl"),
+    indexerUrl: params.get("indexerUrl") ?? chain.indexerUrl,
```

Comment:
since the chain is part of `createStoreSync` (possibly on the `publicClient`), we could check this in there and default it, rather than passing it through in every template